src/applications/personalization/profile-2/helpers.js

```diff
@@ -0,0 +1,47 @@
+import moment from 'moment';
+
+const asyncReturn = (returnValue, delay = 300) =>
+  new Promise(resolve => {
+    setTimeout(() => {
+      resolve(returnValue);
+    }, delay);
+  });
+
+export const asyncFetchMilitaryInformation = () =>
+  asyncReturn({
+    militaryInformation: {
+      serviceHistory: {
+        serviceHistory: [
+          { branchOfService: 'Army', beginDate: moment(), endDate: moment() },
+          {
+            branchOfService: 'Air Force',
+            beginDate: moment(),
+            endDate: moment(),
+          },
+          {
+            branchOfService: 'Marine Corps',
+            beginDate: moment(),
+            endDate: moment(),
+          },
+          {
+            branchOfService: 'Air Force',
+            beginDate: moment(),
+            endDate: moment(),
+          },
+        ],
+      },
+    },
+  });
+
+export const asyncFetchHero = () => {
+  asyncReturn({
+    hero: {
+      userFullName: {
```

Comment:
and this one should just return the object with the name values (no `hero.userFullName` wrapper)